<?xml version="1.0" encoding="UTF-8"?>
<robot generator="Robot 7.2.2 (Python 3.13.2 on win32)" generated="2025-06-16T20:32:24.202478" rpa="false" schemaversion="5">
<suite id="s1" name="RF03 Login" source="c:\Users\henri\Projetos\carest\tests\suites\RF03_Login.robot">
<test id="s1-t1" name="Realizar Login Com Sucesso" line="7">
<kw name="Open Browser" owner="SeleniumLibrary">
<msg time="2025-06-16T20:32:34.726068" level="INFO">Opening browser 'chrome' to base url 'http://localhost:3000'.</msg>
<msg time="2025-06-16T20:32:46.971296" level="WARN">There was an error managing chrome (error sending request for url (https://googlechromelabs.github.io/chrome-for-testing/known-good-versions-with-downloads.json): error trying to connect: dns error: Este host não é conhecido. (os error 11001)); using driver found in the cache</msg>
<arg>${URL}</arg>
<arg>${BROWSER}</arg>
<doc>Opens a new browser instance to the optional ``url``.</doc>
<status status="PASS" start="2025-06-16T20:32:34.725636" elapsed="14.747763"/>
</kw>
<kw name="Go To" owner="SeleniumLibrary">
<msg time="2025-06-16T20:32:49.475584" level="INFO">Opening url 'http://localhost:3000/login'</msg>
<arg>${LOGIN_URL}</arg>
<doc>Navigates the current browser window to the provided ``url``.</doc>
<status status="PASS" start="2025-06-16T20:32:49.474233" elapsed="0.992139"/>
</kw>
<kw name="Wait Until Element Is Visible" owner="SeleniumLibrary">
<arg>${EMAIL_INPUT}</arg>
<doc>Waits until the element ``locator`` is visible.</doc>
<status status="PASS" start="2025-06-16T20:32:50.489208" elapsed="0.414239"/>
</kw>
<kw name="Input Text" owner="SeleniumLibrary">
<msg time="2025-06-16T20:32:50.904584" level="INFO">Typing text 'fuyu@email.com' into text field '//input[@type="email"]'.</msg>
<arg>${EMAIL_INPUT}</arg>
<arg>${EMAIL}</arg>
<doc>Types the given ``text`` into the text field identified by ``locator``.</doc>
<status status="PASS" start="2025-06-16T20:32:50.904089" elapsed="0.181127"/>
</kw>
<kw name="Input Password" owner="SeleniumLibrary">
<msg time="2025-06-16T20:32:51.089371" level="INFO">Typing password into text field '//input[@type="password"]'.</msg>
<msg time="2025-06-16T20:32:51.164665" level="INFO">Temporally setting log level to: NONE</msg>
<arg>${SENHA_INPUT}</arg>
<arg>${SENHA}</arg>
<doc>Types the given password into the text field identified by ``locator``.</doc>
<status status="PASS" start="2025-06-16T20:32:51.088852" elapsed="0.209283"/>
</kw>
<kw name="Click Button" owner="SeleniumLibrary">
<msg time="2025-06-16T20:32:51.299329" level="INFO">Clicking button '//button[contains(., "Entrar")]'.</msg>
<arg>${BTN_LOGIN}</arg>
<doc>Clicks the button identified by ``locator``.</doc>
<status status="PASS" start="2025-06-16T20:32:51.298871" elapsed="0.134190"/>
</kw>
<kw name="Wait Until Element Is Visible" owner="SeleniumLibrary">
<arg>${LOGIN_SUCESSO}</arg>
<arg>timeout=5s</arg>
<doc>Waits until the element ``locator`` is visible.</doc>
<status status="PASS" start="2025-06-16T20:32:51.435440" elapsed="2.228675"/>
</kw>
<kw name="Fechar navegador" owner="setup_teardown" type="TEARDOWN">
<kw name="Close Browser" owner="SeleniumLibrary">
<doc>Closes the current browser.</doc>
<status status="PASS" start="2025-06-16T20:32:53.665588" elapsed="2.644444"/>
</kw>
<status status="PASS" start="2025-06-16T20:32:53.665011" elapsed="2.645893"/>
</kw>
<status status="PASS" start="2025-06-16T20:32:34.724096" elapsed="21.587916"/>
</test>
<status status="PASS" start="2025-06-16T20:32:24.206375" elapsed="32.109039"/>
</suite>
<statistics>
<total>
<stat pass="1" fail="0" skip="0">All Tests</stat>
</total>
<tag>
</tag>
<suite>
<stat name="RF03 Login" id="s1" pass="1" fail="0" skip="0">RF03 Login</stat>
</suite>
</statistics>
<errors>
<msg time="2025-06-16T20:32:46.971296" level="WARN">There was an error managing chrome (error sending request for url (https://googlechromelabs.github.io/chrome-for-testing/known-good-versions-with-downloads.json): error trying to connect: dns error: Este host não é conhecido. (os error 11001)); using driver found in the cache</msg>
</errors>
</robot>
